tcs/heat_exchangers.cpp

```diff
@@ -781,10 +779,6 @@ void NS_HX_counterflow_eqs::solve_q_dot_for_fixed_UA_enth(int hot_fl_code /*-*/,
 			throw(C_csp_exception("Off-design heat exchanger method failed"));
 		}
 	}
-	else
```

Comment:
This control structure isn't needed, it operates in the event that an exception isn't generated, which can be done by simply putting code at the end of the try-catch block